packages/components/src/components/ResizableBox/ResizableBox.tsx

```diff
@@ -52,67 +48,116 @@ const Resizers = styled.div<ResizersProps>(
         ${$maxHeight && `max-height: ${$maxHeight}px;`};
         box-sizing: border-box;
         position: relative;
+        ${
+            $isResizing &&
+            `user-select: none;
+            -webkit-user-select: none;`
+        };
+    `,
+);
 
-        ${$isResizing && `user-select: none;`};
+const handlersCommonStyles = () => css`
+    position: absolute;
+
+    &::after {
+        position: absolute;
+        content: '';
+        display: block;
+        pointer-events: none;
+    }
+`;
+
+const TopHandler = styled.div<ResizerHandlersProps>(
+    ({ $highlightDirection, $zIndex, theme }) => `
+        ${handlersCommonStyles()};
+        width: 100%;
+        height: ${REACTIVE_AREA_WIDTH}px;
+        cursor: ns-resize;
+        top: ${`-${REACTIVE_AREA_WIDTH / 2}px`};
+        ${$zIndex && `z-index: ${$zIndex};`};
```

Comment:
maybe this could be in common styles?

Reply:
I was trying hard to put it there, it was there but wasn't working properly, not sure why. If we leave it here (at least for now) we make sure it works as expected.